src/common/MeasurementInput.jsx

```diff
@@ -93,6 +94,7 @@ class MeasurementInput extends React.Component {
             const state = { pending: false, modelValue };
             if (setValue) {
                 state.value = this.viewValue(modelValue);
+                trackEvent("measurementEntered", "Store: Measurement entered or changed in input field");
```

Comment:
Move this to row 102, after "this.props.onChange(..)". Here it will dispatch an event even when the user doesn't actually change the value